api/sample.env

```diff
@@ -412,6 +412,36 @@ AUTH_SECRET=Change me!
 # sample: SCO_ACCOUNT_RECOVERY_KEY_LIFETIME_MINUTES=10
 # SCO_ACCOUNT_RECOVERY_KEY_LIFETIME_MINUTES=
 
+
+# POLE EMPLOI authentication - Client ID
+# Refer to https://pole-emploi.io/data/api/pole-emploi-connect
+#
+# required for POLE EMPLOI authentication, optional otherwise
```

Comment:
Allez un dernier chipotage pour que ça pareil dans tout le fichier : 

```suggestion
# presence : required for POLE EMPLOI authentication, optional otherwise
```

Qu'on peut rajouter sur les 4 lignes concernées

Reply:
Corrigé ! Ca me semble pas top en anglais malgré le fait qu'il soit référencé
https://www.oxfordlearnersdictionaries.com/definition/english/presence?q=presence